twitter_delete.rb

```diff
@@ -75,6 +75,8 @@ def api_call(method, *args)
   puts "Rate limit exceeded; waiting until #{error.rate_limit.reset_at}"
   sleep error.rate_limit.reset_in
   retry
+rescue Twitter::Error::Forbidden => error
+  puts error.to_s
```

Comment:
This will stop this exception being raised. Could you add a comment (either on GitHub or inline) to explain why?